internal/pkg/cli/storage_init.go

```diff
@@ -848,6 +889,7 @@ Resource names are injected into your containers as environment variables for ea
 	cmd.Flags().BoolVar(&vars.noLSI, storageNoLSIFlag, false, storageNoLSIFlagDescription)
 	cmd.Flags().BoolVar(&vars.noSort, storageNoSortFlag, false, storageNoSortFlagDescription)
 
+	cmd.Flags().StringVar(&vars.rdsServerlessVersion, storageRDSServerlessVersionFlag, "", storageRDSServerlessVersionFlagDescription)
```

Comment:
instead of using `askAuroraServerlessVersion` to set the default, what do you think of setting it here:
```suggestion
	cmd.Flags().StringVar(&vars.rdsServerlessVersion, storageRDSServerlessVersionFlag, defaultServerlessVersion, storageRDSServerlessVersionFlagDescription)
```

Reply:
I fixed it.